_dashboards/im-dashboards/datastream.md

```diff
@@ -41,9 +41,9 @@ To create a data stream, perform the following steps:
 
     ![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams3.png)
 
-1. The **Template details** and **Index alias** sections are read-only. This will inform you of the backing indexes used in the data stream.
+1. The **Inherited settings from template** and **Index alias** sections are read-only. This will inform you of the backing indexes used in the data stream.
```

Comment:
Instead of 'this will inform you of' how about, ".... are read-only, and display the backing indexes that are contained in the data stream." ?

Reply:
Yeah, sounds good, I can make that change.